src/components/Preferences/LocalIdentities.js

```diff
@@ -202,9 +209,36 @@ const LocalIdentities = React.memo(
               Export
             </StyledExport>
           )}
-          <Button label="Remove labels" mode="outline" onClick={onClearAll}>
+          <Button
+            label="Remove labels"
+            mode="outline"
+            onClick={() => setOpened(true)}
+          >
             <IconCross /> Remove all labels
           </Button>
+          <Modal visible={opened} onClose={() => setOpened(true)}>
```

Comment:
We avoid setting functions directly as props because with every render this creates a new function and the Component will see it as a new prop and re-render. We define the handler before and send it as prop:
```
const handleCloes = () => setOpened(false)
// ...

<Modal visible={opened} onClose={handleClose}>
```

Reply:
Don’t forget to cache it :smile: 

```js
const handleClose = useCallback(() => setOpened(false), [])
```